_includes/v21.1/orchestration/kubernetes-limitations.md

```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

Comment:
Aren't we repeating ourselves here? We say no new features are planned. Then with the However we add another no new feature development is currently planned.

Reply:
Also it is missing the verbiage around deprecation announced 6 months earlier

Reply:
I'm not sure what happened here -- I replaced the old text with --- the old text? I'm not sure. It's fixed now.